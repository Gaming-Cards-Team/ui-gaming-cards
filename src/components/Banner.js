import React from 'react';
import banner from "../assets/images/banner.png"
import { BANNER_ALT } from "../util/constant.js"


const Banner = () => (
       <img data-testid='banner'
              className=''
              width= '100%'
              src={banner}
              alt={BANNER_ALT} />

);

export default Banner;