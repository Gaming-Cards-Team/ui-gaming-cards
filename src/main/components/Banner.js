import React from 'react';
import banner from "../resources/banner.png"
import { BANNER_ALT } from "../util/constant.js"


const Banner = () => (
       <img data-testid='banner'
              className=''
              width= '100%'
              src={banner}
              alt={BANNER_ALT} />

);

export default Banner;