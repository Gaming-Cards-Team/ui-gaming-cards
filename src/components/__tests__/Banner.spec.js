import React from 'react';
import Banner from '../Banner';
import {render} from '@testing-library/react';
import renderer from 'react-test-renderer';

describe('Test Banner component', () => {

    it('should validate the component', () => {
        
        const banner = renderer.create(<Banner />).toJSON();
        expect(banner).toMatchSnapshot();
    });

    it('should have a image', () => {
        
        const { getByTestId } = render(<Banner />);
        const imageBanner = getByTestId('banner')

        expect(imageBanner.src).toMatch('banner.png')
    });

});