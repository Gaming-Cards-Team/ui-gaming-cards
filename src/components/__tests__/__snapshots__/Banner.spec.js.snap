// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Banner component should validate the component 1`] = `
<img
  alt="Star Wars Banner"
  data-testid="banner"
  src="banner.png"
  width="100%"
/>
`;
